static/js/browser2/arc_section.tsx

```diff
@@ -35,6 +35,7 @@ interface ArcSectionStateType {
   inLabels: string[];
   outLabels: string[];
   provDomain: { [key: string]: URL };
+  dataFetched: boolean;
```

Comment:
i find prefixing boolean parameters with "is" or "has" helpful for readability, e.g. isDataFetched.

Reply:
done

Reply:
done